Build/Test/IntegrationFrontendTests.xml

```diff
@@ -0,0 +1,29 @@
+<phpunit
```

Comment:
Could we define the constants in `setup()` or in test methods instead of separate the tests in different suites?
What happens with coverage tools like crutinizer & co. by splitting the tests in different suites?

Reply:
Thought about that too, and tried that, but if we don't define this in the phpunit.xml, this is defined by Nimut Testing Framework in its Testing Bootstrap (https://github.com/Nimut/testing-framework/blob/master/src/TestingFramework/Bootstrap/AbstractBootstrap.php#L218) which happens before any PHP code is executed. Because TYPO3 is using a constant (TYPO3_MODE) internally, this can only defined once per PHP process. I had a similar issue with this in Core Testing Framework which isn't solvable in a better way (at least I don't know of) until Core gets rid of TYPO3_MODE code.